src/browser/xhr/xhr.zig

```diff
@@ -438,6 +438,13 @@ pub const XMLHttpRequest = struct {
 
         self.state = .loading;
         self.dispatchEvt("readystatechange");
+
+        if (transfer.getContentLength()) |cl| {
+            if (cl > 100 * 1024 * 1024) {
```

Comment:
Shouldn't we use a common const for this limit accross xhr, scriptmanager and future fetch?